code/game/turfs/floors/desert.dm

```diff
@@ -114,7 +114,7 @@
 			set_light(2)
 			icon = 'icons/turf/floors/desert_water_toxic.dmi'
 		if(0)
-			set_light(1)
+			set_light(0)
```

Comment:
```suggestion
```

Reply:
i cannot remove them, because it will keep light ON